vc_vidyo/indico_vc_vidyo/plugin.py

```diff
@@ -123,6 +124,14 @@ def icon_url(self):
     def _extend_indico_cli(self, sender, **kwargs):
         return cli
 
+    def _event_metadata_postprocess(self, sender, event, data, **kwargs):
+        associations = event.vc_room_associations
+        if associations:
+            description = event.description
+            urls = '\n'.join([assoc.vc_room.vidyo_extension.get_join_url(use_chooser=False) for assoc in associations])
+            return {'description': '\n\n'.join(filter(None, [description, urls]))}
```

Comment:
`filter(None, ...)` is gone in Python 3 - maybe better to not start using such things now?